.github/workflows/snapshot.yml

```
@@ -23,21 +23,22 @@ jobs:
       - name: Install dependencies
         run: yarn --frozen-lockfile
 
-      - name: Check if Prerelease is enabled
-        id: is_pre_enabled
-        uses: andstor/file-existence-action@v1
-        with:
-          files: '.changeset/pre.json'
-
-      - name: Enter prerelease mode
-        if: steps.is_pre_enabled.outputs.files_exists == 'false'
-        run: yarn changeset pre enter next
+      - name: Checkout snapshot branch
+        run: |
+          git checkout -b snapshot
+          git reset --hard ${{ github.ref }}
 
       - name: Create release Pull Request or publish to NPM
         id: changesets
         uses: changesets/action@v1
         with:
-          publish: yarn release
+          version: yarn version-packages --snapshot snapshot
+          publish: yarn release --no-git-tags --snapshot
```

```suggestion
          version: yarn version-packages -- --snapshot snapshot
          publish: yarn release -- --no-git-tags --snapshot
```

Do we need to add option terminators to ensure the additional flags are passed to the `changeset` command?

---

I did test and no extra `--` needed here 👍